pkgs/by-name/bo/bolt-launcher/package.nix

```diff
@@ -29,6 +28,9 @@
   cups,
   systemd,
   buildFHSEnv,
+  enableRS3 ? false,
+  enableHDOSAndRuneLite ? false,
```

Comment:
A bit on the fence about making both of them default to false, as the only part of the package that causes issues is Runescape 3 (additionally, it will be a bit awkward if the user launches the program and ends up not being able to launch any of the games).

Still unsure about it, what was your thought process about disabling both of them?

I think we could either enable `enableHDOSAndRuneLite` by default, or add an assertion to check if both are disabled. That way, the user doesn't end up with a package they cannot use (as this might be confusing if the user doesn't get any messages).

Reply:
Totally agree, any preference about which of those you want to pursue? I imagine most users will likely be looking for HDOSAndRuneLite, but if you think that is too opinionated of an approach we could do the assertion instead?

Reply:
I'd be honestly more enclined towards leaving enableHDOSandRuneLite enabled by default, and disabling enableRS3.

This is the simpler solution that I feel will inconvenience the smallest amount of users. We can also ask on the main PR for more pointers :) but I'd say for now we can just leave it as-is.

Reply:
Yeah that sounds good to me, change made 👍🏻

Reply:
Thanks a lot! Everything seems to work fine. Merging.